com.github.iortcw.metainfo.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Kevin Degeling -->
+
+<component type="desktop">
+  <id>com.github.iortcw</id>
+  <name>IO Return to Castle Wolfenstein</name>
+  <summary>Game engine for Return to Castle Wolfenstein</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+
+  <developer_name>iortcw team</developer_name>
+  <update_contact>eonfge@kevindegeling.nl</update_contact>
+
+  <url type="homepage">https://github.com/iortcw/iortcw</url>
+
+  <launchable type="desktop-id">com.github.iortcw.desktop</launchable>
```

Comment:
The catch here is that it either should point to the launcher you consider main or at least one launcher should match the full ID of the app, otherwise `appstream-compose` does not know where to look for icon information. This file as it is simply does not exist.

Reply:
I've settled on making the singleplayer mode, the primary mode. I already had to compromise a bit, so this works. Just so you know, the original Windows Desktop release from 2001 also had to launchers ;)